androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/AlertCard.kt

```diff
@@ -51,6 +52,9 @@ fun AlertCard(
         when (spec) {
             AlertCardSpec.Downstream -> formattedAlert.downstreamEffect
             AlertCardSpec.Elevator -> alert.header ?: formattedAlert.effect
+            AlertCardSpec.Delay ->
+                formattedAlert.cause?.let { stringResource(R.string.delays_due_to_cause, it) }
+                    ?: stringResource(R.string.delays_unknown_reason)
```

Comment:
As I recall, "X between Y and Z due to A" (or "X due to A" or "X between Y and Z") is the way we're interested in presenting a lot of alerts, and delays are just the first alert that we're handling that way. I was about to suggest making this more generic, but since calculating the boundaries of the alert is the actual hard part and delays generally apply to the whole route, it may not be worth being slightly more generic now if we're still going to have to do all the hard work later.

Reply:
Yeah that is true, happy to hold off on making it more generic for now 😄